src/SourceLocator/SourceStubber/SourceStubber.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\SourceLocator\SourceStubber;
+
+use ReflectionClass as CoreReflectionClass;
+use ReflectionFunction as CoreReflectionFunction;
+
+interface SourceStubber
+{
+    public function getClassStub(CoreReflectionClass $classReflection) : ?string;
+
+    public function getFunctionStub(CoreReflectionFunction $functionReflection) : ?string;
```

Comment:
Please document this. The alternative is renaming to `generateFunctionStubCode` to make it more obvious. The nullable return type is also unclear when just looking at this interface